deploy/start.sh

```diff
@@ -16,6 +16,12 @@ else
     echo "  \"azureTenantId\": \"${REACT_APP_AZURE_TENANT_ID}\"," >> $envfile
 fi
 
+if [[ -z "${REACT_APP_ENABLE_RBAC}" ]]; then
```

Comment:
Please update docc [https://github.com/linkedin/feathr/blob/main/docs/dev_guide/build-and-push-feathr-registry-docker-image.md](https://github.com/linkedin/feathr/blob/main/docs/dev_guide/build-and-push-feathr-registry-docker-image.md), the section Test SQL registry + RBAC needs update

Reply:
Updated